mongostat/main/mongostat.go

```diff
@@ -237,7 +237,10 @@ func main() {
 	}
 
 	for _, v := range seedHosts {
-		stat.AddNewNode(v)
+		if err := stat.AddNewNode(v); err != nil {
```

Comment:
Good catch. Can you file a ticket to add errcheck to our CI process? We shouldn't be forgetting errors anywhere.

Reply:
Filed https://jira.mongodb.org/browse/TOOLS-2266